shell_scripts/external_router_vms/tempest_conf.py

```diff
@@ -15,12 +15,10 @@
 
 AUTH_TEMPLATE="""
 [auth]
-# un-comment for juju queens
-#admin_domain_name = admin_domain
 tempest_roles = member
 admin_username = admin
 admin_project_name = admin
-admin_domain_name = Default
+admin_domain_name = %(admin_domain_name)s
```

Comment:
Note: we'll have to update run_tests.py, as it's currently modifying this part of the template for queens (searches for the string 'admin_domain_name = Default' and deletes that line).